apps/server/src/utils/coerceType.ts

```diff
@@ -40,3 +56,19 @@ export function coerceNumber(value: unknown): number {
   }
   return parsedValue;
 }
+
+/**
+ * @description Converts a value to a colour if possible, throws otherwise
+ * @param {unknown} value - Value to be converted to a colour.
+ * @returns {string} - The converted value as a string.
+ * @throws {Error} Throws an error if the value is null or undefined.
+ */
+export function coerceColour(value: unknown): string {
+  if (value == null) {
+    throw new Error('Invalid value received');
+  }
+  if (typeof value === 'string') {
+    value = value.toLocaleLowerCase(); //Red will not work but red will
+  }
+  return String(value);
+}
```

Comment:
I dont think this coerces into a colour, it just forces a value to be defined and to be a lower case string, is that not correct?
If we were to pass in a user value 000, we would just get "000" back

Also, 

- remember not to mutate the things we pass in
- in the current code, if the value is already a string it will still be coerced to String on return